content/writing/pgvector-vs-qdrant.md

```diff
@@ -1,5 +1,5 @@
 +++
-title =  "1M OpenAI Benchmark: PGVector vs VectorDB (Qdrant)"
+title =  "Qdrant vs pgvector - Results from the 1M OpenAI Benchmark"
```

Comment:
There are other benchmarks on Qdrant vs pgvector, which are not 1M, or not OpenAI embeddings e.g. the official ann-benchmarks.com themselves — that is why it was first. Do you still think this is better?